### PR TITLE
Adding GitHub Actions

### PR DESCRIPTION
**Pull request by priyansh19**
- Added Greetings.yml
- Added Label.yml
- Added Stale.yml

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Congratulations! You have created an Issue :+1: Our maintainers will soon revert back to you'
```
Comment (Bhupesh-V) [inline .github/workflows/greetings.yml]:
```suggestion
        issue-message: 'Thanks for reporting this! Our maintainers will soon revert back to you'
```

**Comment by priyansh19** [comment]
@xandao6 @rpotter12 .. review this pr #213

**Comment by Bhupesh-V** [comment]
> @priyansh19 I don't know anything about these but looks good. Wait for others approval who is having knowledge about these scripts.

I think it would be fine to merge these looks :ok:

**Comment by priyansh19** [comment]
@rpotter12 please merge this PULL Request all mentors have reviewed this 😊

**Pull request merged**